blazor/rating/appearance.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Appearance in Blazor Rating Component | Syncfusion
+description: Checkout and learn here all about Appearance with Syncfusion Blazor Rating component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Rating
+documentation: ug
+---
+
+# Appearance in Blazor Rating Component
+
+You can also customize the appearance of rating control.
+
+## Items count
+
+You can specify the number of rating items using the [ItemsCount](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_ItemsCount) property.
+
+Below example demonstrates the Items Count of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating ItemsCount="8"></SfRating>
+
+```
+
+![Blazor Rating Component with Items Count](images/blazor-rating-itemscount.png)
```

Comment:
Ensure that you have committed all the images. The above image is not in the committed files